cms/djangoapps/contentstore/management/commands/export_content_library.py

```diff
@@ -0,0 +1,54 @@
+"""
+Script for exporting a content library from Mongo to a tar.gz file
+"""
+from __future__ import print_function
+import os
+
+from django.core.management.base import BaseCommand, CommandError
+from opaque_keys import InvalidKeyError
+from opaque_keys.edx.keys import CourseKey
+
+from cms.djangoapps.contentstore import tasks
+
+
+class Command(BaseCommand):
+    """
+    Export the specified content library into a directory.  Output will need to be tar zxcf'ed.
+    """
+    help = 'Export the specified content library into a directory'
+
+    def add_arguments(self, parser):
+        parser.add_argument('library_id')
+        parser.add_argument('output_path')
+
+    def handle(self, *args, **options):
+        """
+        Given a content library id, and an output_path folder.  Export the
+        corresponding course from mongo and put it directly in the folder.
+        """
+        module_store = tasks.modulestore()
+        try:
+            library_key = CourseKey.from_string(options['library_id'])
+        except InvalidKeyError:
+            raise CommandError("Invalid course_key: '%s'." % options['library_id'])
```

Comment:
This string should say "Invalid library ID:"

Also please use `.format()` instead of `%` throughout. There seems to be a mix of the old `%` and the new `format()` at the moment.

Reply:
👍